nav2_util/src/geometry_utils.cpp

```diff
@@ -0,0 +1,29 @@
+// Copyright (c) 2019 Intel Corporation
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#include "nav2_util/geometry_utils.hpp"
+#include "tf2_geometry_msgs/tf2_geometry_msgs.h"
+
+namespace nav2_util
+{
+
+geometry_msgs::msg::Quaternion
+geometry_utils::orientationAroundZAxis(double angle)
+{
+  tf2::Quaternion q;
```

Comment:
just being knitpicky, but why isnt this just an inline function?

Reply:
Sure, I'll update to make it inline.